sys/include/ufs/inode.h

```diff
@@ -59,7 +59,7 @@ typedef struct ufs2_dinode ufs2_dinode;
  * exclusive.
  */
 struct inode {
-	TAILQ_ENTRY(inode) i_nextsnap; /* snapshot file list. */
+	//TAILQ_ENTRY(inode) i_nextsnap; /* snapshot file list. */
```

Comment:
will this ever be used again? If it might get turned on again maybe a TODO is a good idea? If never again, remove it?

Reply:
It'll be used.  When we enable the code that uses it, it'll be clear that it's needed.  I can add a todo though, just in case anyone else removes it.

Reply:
We have an opened issue for tailq, references could be there and mentioned in a comment.

Reply:
The intention is to replace it with non-macro lists, as in issue #605